ModernWallet/Classes/ViewController/Settings/SettingsTableViewController.swift

```diff
@@ -158,6 +158,10 @@ fileprivate extension ObservableType where Self.E == SettingsTableViewController
             }
             else if let indexPath = vc.tableView.indexPathForSelectedRow {
                 vc.tableView.deselectRow(at: indexPath, animated: true)
+                if rowInfo.title == "REFUND" {
```

Comment:
Please make that check robust so that it would work with other language than English.